src/commands/auth/store-token/store.ts

```diff
@@ -20,7 +20,7 @@ export async function storeToken(token: string) {
 
     console.log("Token and user info stored securely using system keychain.");
   } catch (error) {
-    console.error("Error storing token in system keychain.");
+    console.warn("Error storing token in system keychain. Please check 'configuring-keytar-on-linux.md' on the anyflow-cli npm package page.");
```

Comment:
From a UX perspective, I would prefer to leave it to use the fallback if keytar fails instead of guiding the user to read a docs. There's no problem in using local file.